src/main/app/src/Components/Landscape/Modals/Item/Item.tsx

```diff
@@ -413,6 +421,13 @@ const Item: React.FC<Props> = ({ fullyQualifiedItemIdentifier, small }) => {
                 </div>
               ) : null}
 
+              {networks ? (
+                <div className='networks'>
+                  <Typography variant={'h6'}>Networks</Typography>
```

Comment:
We should start to use frontend mapping where possible

Reply:
Should frontend mapping only append to "networks" or also to "frameworks", "labels", etc. for consistency?
If yes, Robert and I would suggest to create a dedicated ticket.
To be discussed.

Reply:
👍 a dedicated ticket sounds reasonable